gangplank/internal/spec/jobspec.go

```diff
@@ -187,7 +193,7 @@ func (r *Repo) Writer(path string) (string, error) {
 	closer := func() error { return nil }
 	var dataR io.Reader
 	if r.URL != nil && *r.URL != "" {
-		resp, err := http.Get(*r.URL)
+		resp, err := httpGet(*r.URL)
```

Comment:
This is still not dealing with the fact that non HTTP 200 Responses do not throw an error, thus not catching potential bad repos.

Reply:
For non-http, the response we'll get an err from the `http.Get()` func.

Reply:
Yeah, for non-http responses, not for HTTP-Error Codes. So a HTTP 404 will not throw an error and is also not guaranteed to not return an error page, instead of nothing. Therefore the bad repo test in this configuration would fail with an actual web-server, as we would accept the 404 Error Page as the Repository, since the Status Code is ignored.

Reply:
Fixedup.